samples/generated/static-spa/README.md

```diff
@@ -0,0 +1,45 @@
+# static-spa sample
+
+This sample demonstrates using `@okta/okta-auth-js` within a SPA (single-page application). By default the app server runs at `http://localhost:8080`. The callback redirect URI is `http://localhost:8080/implicit/callback`
```

Comment:
use `/login/callback`?

Reply:
Until /login/callback is the default from the dev console, I prefer not to recommend it.

Do we have a ticket to make that the default?  If not, I'll add one

Reply:
per the ticket, I don't see any for developer console change, also which team to assign the ticket to? 
But looks like Matt has already started changing the callback URL for sample repos.

Reply:
Just created a ticket for the change, we'll chase down who is responsible. (probably will fall to us until the new team is on board).

Matt has already started, yes, but evangelism has a few options, and while I like and will champion `/login/callback`, I still think the best option is to follow the default.  Once that changes I'm happy to have our samples follow suit.

Reply:
I think it will fall to us. Also we can add multiple callbacks as the default if we like.